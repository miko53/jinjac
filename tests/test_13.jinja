{{ "%s - %s"|format("Hello?", "Foo!") }}
{{ "%d - %.8d"|format(15, 15684) }}
{{ "%.12f - %f"|format(121.651, 0.1684) }}
