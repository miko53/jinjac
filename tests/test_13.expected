Hello? - Foo!
15 - 00015684
121.651000000000 - 0.168400
